About.html

```diff
@@ -0,0 +1,160 @@
+
+ <!-- About page -->
+ <!DOCTYPE html>
+ <html lang="en">
+ <head>
+  <link rel="stylesheet" href="about.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.cdnfonts.com/css/lato-light" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+  <title>About-page</title>
+ </head>
+ <body>
+  <body >
+    <div class="nav-top">
+        <ul class="">
+       
+         <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+    <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+    <li class="top"><a href="#">English</a></li>
+    <li><a href="#">My page</a></li>
+    <li><a href="#">Logout</a></li>
+        
+        </ul>
+        </div>
+<section class="home-page"> 
+    <header class="hearer-second">
+       
+        <a class="logo" href="/home.html"><img src="logos/logo.png" alt="home page logo"></a>
+         <div class="menu-toggle">
+           <img class="menu-img" src="images/menu.png" alt="">
+          </div>
+         </div>
+         <nav>
+         <button class="close-button">&times;</button>
+         <div>
+           <ul class="links">
+            <li class="about"><a href="/About.html">About</a></li>
```

Comment:
`[OPTIONAL]` 
- [ ] KIndly note that once you are on the `About` page, your nav item `About` should be changed to `Home` so that the user can navigate back to the homepage.